docs/deployment/docker-swarm.md

```diff
@@ -35,8 +35,26 @@ $ git clone https://github.com/openfaas/faas && \
   ./deploy_stack.sh
 ```
 
-!!! note
-    If you want to try newer features you can checkout the `master` branch, but we do not recommend that for first-time users.
+!!! info
+    Basic authentication is now enabled by default for your protection. If you need to disable it pass the flag --no-auth to the ./deploy_stack.sh command above.
+
+### 2.1 Store your admin credentials
+
+The default configuration will create a username and password combination for you:
+
+```
+Attempting to create credentials for gateway..
+...
+[Credentials]
+ username: admin
+ password: <some_hash_secret>
+ echo -n <some_hash_secret> | faas-cli login --username=admin --
+password-stdin
+```
+
+Run the command as you see it in your console, do not copy/paste the login command.
```

Comment:
Is the newline splitting --password-stdin intended?

Is the instruction following the code block meant to mean, _"do not use the example command shown above, instead use the command as shown in your console"_?